roles/netapp-trident/tasks/main.yml

```diff
@@ -7,153 +7,23 @@
     dest: /root
     remote_src: yes
 
-- name: Create Trident namespace (ubuntu)
-  k8s:
-    state: present
-    kind: Namespace
-    name: trident
-    wait: yes
+- name: Copy namespace config file to control node
+  template:
+    src: namespace.j2
+    dest: /root/trident-installer/namespace.yaml
```

Comment:
Can we replace /root-rident-installer with a variable rather than hardcoding this path place?

Also, we've been trying to keep most of the deepops files in a cenral place. We define deepops_dir in all.yaml and by default that is /opt/deepops. If there is a bunch of existing Trident documentation that references /root/trident-installer, I think it makes sense to leave it there. But I'd still replace this directory with a variable and reference that in all the places that follow.

Reply:
@supertetelman - I'll replace this hardcoded path with a variable, and I'll incorporate the existing deepops_dir variable. Stay tuned for an updated PR.

Reply:
@supertetelman - The PR is updated, but I need to validate the installation process with the changes. I'll ping back after I validate it (will likely be tomorrow).

Reply:
@supertetelman - I validated a clean installation with the new changes.